extensions/cli/debug/src/main/java/mil/nga/giat/geowave/cli/debug/MinimalFullTable.java

```diff
@@ -122,4 +122,11 @@ else if (storeType.equals(HBaseDataStore.TYPE)) {
 					"full scan for store type " + storeType + " not implemented.");
 		}
 	}
+
+	@Override
+	public Object computeResults(
+			OperationParams params ) {
+		// TODO Auto-generated method stub
+		return null;
```

Comment:
problem (albeit merely debug commands, but still seems relatively straightforward to fix)